### DISCUSSION TITLE
paperless-ngx - how to access administration commands (export, import, etc.)

### DISCUSSION THREAD

**Discussion by PozoSer** [tteck/Proxmox#1786] [Help Needed] [answered]
Hello

I'm referring to these useful utilities: https://docs.paperless-ngx.com/administration/#bare-metal-route (document_export, document_import, retagging tools, etc.). According to paperless-ngx documentation, commands should be executed from cd /path/to/paperless/src (which I guess in the scripts is set up as /opt/paperless/src). However, I've tried some commands with no luck. If I run document_exporter i get "command not found"

Any help?

**Comment by tteck** [comment]
What is the specific command that you are executing?

**Reply by PozoSer** [reply]
document_importer "path"

**Reply by PozoSer** [reply]
The python script to import seem to be on /opt/paperless/src/documents/management/commands/, but running it from that folder does not work either.

**Reply by tteck** [reply, marked as answer]
Are you executing the command as specified in the documentation?

https://docs.paperless-ngx.com/administration/#management-commands

```
cd /opt/paperless/src
python3 manage.py document_importer <source>
```

**Comment by PozoSer** [comment]
Ohh I see. I misunderstood the command. I marked the question as solved. 

Two more questions regarding paperless if you are so kind:

1. Am I correct that I still need to set up a redis database on another container/docker for the scheduling taks to work? I saw no redis database configured on /opt/paperless/paperless.conf and the official paperless manual (https://docs.paperless-ngx.com/configuration/#redis-broker) says that _redis is required for processing scheduled tasks such as email fetching, index optimization and for training the automatic document matcher._
2. Is paperless configured as using postgresql as default database (should be according to paperless documentation I want to check before I start debugging why my document_import failed due to a sqlite error)?

Thanks!

**Reply by tteck** [reply]
My primary focus revolves around the development of application LXCs, and I'm not specialized in providing support for operational issues that may arise with these applications. 

2. https://github.com/tteck/Proxmox/discussions/1716

**Reply by tteck** [reply]
redis is installed but commented out in the `paperless.conf`

**Comment by tteck** [comment]
Modified the script to incorporate Redis and PostgreSQL, while also introducing an option to include Adminer during installation.

**Reply by tteck** [reply]
![Screenshot 2023-09-11 3 35 45 PM](https://github.com/tteck/Proxmox/assets/21342670/9e8f103d-8f8d-4a16-9a65-048a03c2760e)

**Reply by PozoSer** [reply]
You're truly amazing. Thanks!
